book/src/zcash/ywallet-demo.md

```diff
@@ -128,36 +131,51 @@ run):
 RUST_LOG=debug cargo run --bin server
 ```
 
-In the second one, the Coordinator, run (in the same folder where key generation
-was run):
+### Registering users
+
+In order to interact with the server, you will need to register users. For this
+guide we will need two. In a new terminal, run the following command for user
+"alice" (replace the password if you want):
 
 ```
-cargo run --bin coordinator -- -C redpallas --http -r -
+curl --data-binary '{"username": "alice", "password": "foobar10", "pubkey": ""}' -H 'Content-Type: application/json' http://127.0.0.1:2744/register
```

Comment:
MIght be good to explain what the pubkey parameter is about and why it's an empty string here

Reply:
Done in 412dd015